common/Authorization.cpp

```diff
@@ -80,7 +80,7 @@ void Authorization::authorizeRequest(Poco::Net::HTTPRequest& request) const
             break;
         }
         default:
-            LOG_TRC("No HTTP Authorization type detected. Assuming no authorization needed. "
+            LOG_ERR("No HTTP Authorization type detected. Assuming no authorization needed. "
```

Comment:
Interesting. It's hard to know if there is no http authorization intentionally or it's a bug. If it's intentional, then this will be a very annoying ERR that will fill the logs (it will be logged for each and every URI request we process!). But if it's a bug, it's a helpful hint, although without the authorization documents won't load anyway, so it's not going to be unnoticed.

I think we won't be helpful if we make this an ERR, because there is no requirement for http authorization (this error will make it a requirement, implicitly and without documentation, because there is no way to silence this error without having http authorization). So, we either make a requirement and change this to ERR, or, if it's nor required, then we can change this to INF, so that it's more visible when troubleshooting.